docs/source/user/installing.rst

```diff
@@ -15,12 +15,12 @@ Ubuntu and Debian users will use ``sudo apt install guake``.
 Fedora
 ------
 
-Tested in 29:
+Tested in Fedora 34:
 We can use dnf to install it.
 
 .. code-block:: bash
 
-    $ sudo dnf install -y guake
+    $ sudo dnf install guake
```

Comment:
Haven't used fedora in a long while, but isn't the -y for auto yes? Is there a reason why we don't want that?

Reply:
I changed this because I think it is a better practice to let users review what they are installing. For example, packages may have optional dependencies ( called `weak dependencies` in Fedora)  and they may want to opt out of installing them.

Reply:
eh sure alright, not that big an issue with guake though